migrations/versions/ec8f94139b02_codify_workspace_name_and_message_id_in_.py

```
@@ -0,0 +1,298 @@
+                WITH batch AS (
+                    SELECT id
+                    FROM "{schema}".queue
+                    WHERE payload ? 'workspace_name'
+                    ORDER BY id
+                    LIMIT :batch_size
+                )
+                UPDATE "{schema}".queue q
+                SET payload = q.payload - 'workspace_name'
+                FROM batch
+                WHERE q.id = batch.id
+                """
```

Ruff (0.14.3)

Possible SQL injection vector through string-based query construction

(S608)